### PR TITLE
Add flake8-bugbear B907 to ignore list

### PR DESCRIPTION
**Pull request by Secrus**
### Description

This adds newly introduced flake8-bugbear `B028` rule to ignore list
### Checklist - did you ...

- [ ] Add an entry in `CHANGES.md` if necessary?
- [ ] Add / update tests if necessary?
- [ ] Add new / update outdated documentation?

**None of those are necessary here.**

### CODE REVIEW

**Review by JelleZijlstra — APPROVED** [review]

**Comment by cooperlees** [comment]
Due to user dislike - New flake8-bugbear released with this check moved to optional B907 ...

File: .flake8 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [flake8]
 # B905 should be enabled when we drop support for 3.9
-ignore = E203, E266, E501, W503, B905
+ignore = E203, E266, E501, W503, B905, B028
```
Comment (cooperlees) [inline .flake8:3]: Maybe we should just move forward flake8-bugbear version to 23.1.17 + disable B907
```suggestion
ignore = E203, E266, E501, W503, B905, B907
```

**Review by cooperlees — COMMENTED** [review]
Sorry for the churn.

File: .flake8 (hunk 1 above)
Comment (JelleZijlstra) [reply .flake8:3]: flake8-bugbear isn't pinned for us, so I think we're automatically using the latest.

Comment (Secrus) [reply .flake8]: Done

**Pull request merged**